config/releases.exs

```diff
@@ -1 +1,16 @@
 import Config
+
+alias Skate.SecretsManager
+
+Application.ensure_all_started(:poison)
+Application.ensure_all_started(:hackney)
+Application.ensure_all_started(:ex_aws)
+Application.ensure_all_started(:ex_aws_secretsmanager)
+
+config :skate,
+  secret_key_base: SecretsManager.fetch!("SECRET_KEY_BASE")
+
+config :ueberauth, Ueberauth.Strategy.Cognito,
+  client_secret: SecretsManager.fetch!("COGNITO_CLIENT_SECRET")
+
+config :skate, SkateWeb.AuthManager, secret_key: SecretsManager.fetch!("GUARDIAN_SECRET_KEY")
```

Comment:
Why this as opposed to `{:secret, "GUARDIAN_SECRET_KEY"}` and let `Application.runtime_config` take care of it? 

Does this config file get run when devloping locally, and would that be a problem?

If we can load config at runtime in this file, should we replace the whole `runtime_config` system with `System.get_env` calls in this file?

Reply:
This config file does not get run when developing locally, and that's the reason for these to be here—they are only needed in prod and not in dev.

Reply:
I think it'd be possible to fetch it from the secrets manager in prod, and not in dev, and have a consistent way of specifying all the config/secrets if it was like this:
```exs
#config.exs
secret_key: nil
```
```exs
#prod.exs
secret_key: {:secret, "GUARDIAN_SECRET_KEY"}
```
```exs
#releases.exs
#no runtime configuration here, it's all prod.exs and Application.runtime_config()
```

Reply:
Personally, that feels more confusing to me—the key being defined in prod but undefined elsewhere seems less clear.

Reply:
`config.exs` already has that line.

The goal with this suggestion would be consolidate all the places where prod checks the secrets manager (currently `releases.exs`, `prod.exs/Application.runtime_config`, and `endpoint.ex`) into just one place (`prod.exs/Application.runtime_config`).

Dev configuration wouldn't change.

Reply:
Oh sorry, I was confused because that setting isn't on `:skate`, it's on `SkateWeb.AuthManager`. The application `load_runtime_config` is just specific to the skate app.

Reply:
Ah, I didn't realize that. Should we make `load_runtime_config` able to work on all apps? We could write the list of keys as `[{:skate, :api_url}, {SkateWeb.AuthManager, :secret_key}]`

Reply:
Yeah, I could see that, but I think that's outside the scope of this ticket. This change isn't really making things any more or less consolidated. That said as I think about it the amount of variation is pretty limited and the SecretsManager code is all encapsulated, so I'm not sure how much extra effort on `load_runtime_config` would be worth it…